gui/mod-manager.lua

```diff
@@ -401,6 +402,120 @@ function ModmanageScreen:init()
     }
 end
 
+ModlistMenu = defclass(ModlistMenu, widgets.Window)
+ModlistMenu.ATTRS {
+    view_id = "modlist_menu",
+    frame_title = "Active Modlist",
+    frame_style = gui.WINDOW_FRAME,
```

Comment:
```suggestion
```

Reply:
you don't need to specify the `frame_style` for a `widgets.Window` unless you're making a non-standard Window